build.gradle

```diff
@@ -57,6 +57,7 @@ dependencies {
     implementation "org.grails:grails-datastore-gorm:$datastoreVersion"
 
     compileOnly "jakarta.servlet:jakarta.servlet-api:$servletApiVersion" // Provided
+    compileOnly "jakarta.persistence:jakarta.persistence-api:$jakartaPersistenceVersion"
```

Comment:
Why is this dependency added? I can't find any usage of it it in the project.

Reply:
Without it the build was failing all of the sudden: https://github.com/grails/grails-plugin-converters/actions/runs/10818461245/job/30014131537

```
> Task :compileGroovy


FAILURE: Build failed with an exception.

> Task :compileGroovy FAILED
* What went wrong:
2 actionable tasks: 2 executed
Execution failed for task ':compileGroovy'.
> jakarta/persistence/Entity
```

Reply:
Looking at the `compileClasspath` from that run (https://ge.grails.org/s/2q4bac7ymiggk/dependencies?toggled=W1swXSxbMCwyXSxbMCwyLFsyOTNdXV0) it seems `org.grails:grails-datastore-gorm:7.3.4` was used.

Reply:
Did you try to run the build without jakarta-persistance-api?

Reply:
I did locally and reproduced this issue and then resolved it initially with `compileOnly "jakarta.persistence:jakarta.persistence-api:$jakartaPersistenceVersion"`

In retesting through, I have found that `grailsGradlePluginVersion=7.0.0-SNAPSHOT` also resolved the issue, so I will remove the extra dependency which is not need with the new Gradle plugin version.